### PR TITLE
Allow KUBE_ROOT to be defined externally

### PR DESCRIPTION
**Pull request by pditommaso**

### CODE REVIEW

**Comment by pditommaso** [comment]
I signed it!

File: cluster/kubectl.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ set -o pipefail
 # echo "-=-=-=-=-=-=-=-=-=-="
 
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=${KUBE_ROOT:-dirname "${BASH_SOURCE}")/..}
```
Comment (brendandburns) [inline cluster/kubectl.sh]: does this work?  I think you need ticks around the command?

Comment (pditommaso) [inline cluster/kubectl.sh]: Yes, there was a typo. Thanks for pointing it out.

**Comment by brendandburns** [comment]
@k8s-bot ok to test

Thanks for the PR.

**Pull request merged**